config/urls.py

```diff
@@ -48,6 +55,12 @@
     path("admin/", admin.site.urls),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
 
+if settings.SENTRY_DEBUG_URL_ENABLED:
+    # url is toggled via the SENTRY_DEBUG_URL_ENABLED .env var
+    urlpatterns = [
+        path("sentry-debug/", trigger_error),
+    ] + urlpatterns
+
```

Comment:
I know this is only designed to be turned on for testing but I'm not comfortable with code that is designed to intentionally throw errors left in production.

There are other ways we can test the connection so I feel this would be better left out.